### PR TITLE
schema update

### PR DESCRIPTION
**Pull request by vhvb1989**
complement of: https://github.com/Azure/azure-dev/pull/3155

To be merged on release-day to update schema

### CODE REVIEW

File: schemas/v1.0/azure.yaml.json [hunk 1]
```diff
@@ -239,6 +239,28 @@
...
+                },
+                "secrets": {
+                    "type": "array",
+                    "title": "Optional. List of azd environment variables to be used in the pipeline as secrets.",
+                    "description": "If variable is found on azd environment, it is set as a secret for the pipeline.",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "additionalVariablesAsSecrets": {
```
Comment (jongio) [inline schemas/v1.0/azure.yaml.json:259]: This seems like overkill to me to add this feature.  I also don't think the name matches the description/functionality.

The name looks like "add these" the description states "add all vars as secrets"

File: schemas/v1.0/azure.yaml.json [hunk 2]
```diff
@@ -239,6 +239,28 @@
...
+                    "type": "array",
+                    "title": "Optional. List of azd environment variables to be used in the pipeline as variables.",
+                    "description": "If variable is found on azd environment, it is set as a variable for the pipeline.",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "secrets": {
+                    "type": "array",
+                    "title": "Optional. List of azd environment variables to be used in the pipeline as secrets.",
```
Comment (jongio) [inline schemas/v1.0/azure.yaml.json:253]:
```suggestion
                    "title": "Optional. List of azd environment variable names to be used in the pipeline as secrets.",
```

File: schemas/v1.0/azure.yaml.json [hunk 3]
```diff
@@ -239,6 +239,28 @@
                         "github",
                         "azdo"
                     ]
+                },
+                "variables": {
+                    "type": "array",
+                    "title": "Optional. List of azd environment variables to be used in the pipeline as variables.",
```
Comment (jongio) [inline schemas/v1.0/azure.yaml.json:245]:
```suggestion
                    "title": "Optional. List of azd environment variable names to be used in the pipeline as variables.",
```

File: schemas/v1.0/azure.yaml.json (hunk 1 above)
Comment (vhvb1989) [reply schemas/v1.0/azure.yaml.json:259]: name is not final.  See: https://github.com/Azure/azure-dev/pull/3155#issuecomment-1876179447

**Comment by vhvb1989** [comment]
moved it to: https://github.com/Azure/azure-dev/pull/3330/files

**Pull request closed without merging**